metrotranscom/doorway
# feat: add professional partners section

# Pull Request Template

## Issue Overview

This PR addresses #257


## Description

Adds Professional partners tab with 2 new pages.

## How Can This Be Tested/Reviewed?

To show section in navigation you need to add `SHOW_PROFESSIONAL_PARTNERS=TRUE` to .env.local.
Then it will show on right side of navigation. 
There should be 2 pages with content from [here](https://docs.google.com/document/d/12FEQPqAgH3t96IviNgPxox-Bj3j0IqkCdfWYa7h_MYs/edit)


## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
